src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs

```diff
@@ -11,7 +11,7 @@ public static partial class RuntimeInformation
     {
 #if uapaot
         private const string FrameworkName = ".NET Native";
-#elif netfx || win8
+#elif win8
```

Comment:
You are right in your comment, looks like `win8` can be removed as well, but I'm fine with scoping your PR to just netfx for now, and do another pass later for other unused configurations.